config/application.rb

```diff
@@ -36,6 +37,15 @@ class Application < Rails::Application
 
       config.paths['log']             = "#{Msf::Config.log_directory}/#{Rails.env}.log"
       config.paths['config/database'] = [Metasploit::Framework::Database.configurations_pathname.try(:to_path)]
+
+      case Rails.env
+      when "development"
+        config.eager_load = false
+      when "test"
```

Comment:
Since the bodies of these `when` blocks are identical, they should be combined, like so:
```ruby
case Rails.env
when "development", "test"
  config.eager_load = false
when "production"
  config.eager_load = true
end